plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/credentials/ToolkitAuthManager.kt

```diff
@@ -410,6 +432,8 @@ private fun recordAddConnection(
     }
 }
 
+private var hasSeenFirstNetworkError = false
```

Comment:
Instead of using a global variable here, do we want to use persistent storage. Im wondering about the case where two different services trigger the same function around the same time, will this notiification be shown twice then?